libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/NavigationViewModel.java

```diff
@@ -215,11 +219,21 @@ private void initNavigationLocationEngine() {
   }
 
   private void initLanguage(NavigationUiOptions options) {
-    language = options.directionsRoute().routeOptions().language();
+    RouteOptions routeOptions = options.directionsRoute().routeOptions();
+    if (routeOptions != null) {
+      language = routeOptions.language();
+    } else {
+      language = localeUtils.inferDeviceLanguage(getApplication());
```

Comment:
If we decide updating the code as suggested in https://github.com/mapbox/mapbox-navigation-android/pull/1229/files#r212031761 I believe we could update this block of code the same fashion.